Ladybird/WebContentView.cpp

```diff
@@ -1093,3 +1093,8 @@ void WebContentView::notify_server_did_finish_handling_input_event(bool event_wa
     //        we don't need to do anything here. But we'll need to once we start asking web content first.
     (void)event_was_accepted;
 }
+
+void WebContentView::notify_server_did_get_accessibility_tree(DeprecatedString const&)
```

Comment:
Please merge this commit into "LibWebView+WebContent: Expose the Accessibility Tree to Other Processes" so that the system will always build OK after each commit. (This makes doing a `git bisect` a lot less painful.)

Also, please resolve any feedback comments you've dealt with already. :^)